BOJ/solved.ac_class/Class02/1978.소수 찾기/sAp00n.py

```diff
@@ -0,0 +1,32 @@
+from sys import stdin
+
+
+def sieve_of_eratosthenes(Maximum):
+    numbers = {}
+    for number in range(2, Maximum + 1):
+        numbers[number] = True
+    root_of_max = int(Maximum ** 0.5)
+    for i in range(2, root_of_max + 1):
+        if numbers[i]:
+            n = 2
+            while i * n <= Maximum:
+                numbers[i * n] = False
+                n += 1
```

Comment:
```suggestion
        if not numbers[i]: break
        n = 2
        while i * n <= Maximum:
            numbers[i * n] = False
            n += 1
```
depth는 최대한 얕게 유지해주세요!